runtime/rasdump/dmpsup.c

```diff
@@ -1095,14 +1095,12 @@ pushDumpFacade(J9JavaVM *vm)
 		/* Install new facade */
 		vm->j9rasDumpFunctions = (J9RASdumpFunctions *)queue;
 
-		/* Note that we need to do the same check in popDumpFacade()
-		 *
-		 * According to the port library SIGABRT (J9PORT_SIG_FLAG_SIGABRT) is an ASYNC signal, so you'd think we should be checking for J9PORT_SIG_OPTIONS_REDUCED_SIGNALS_ASYNCHRONOUS here.
-		 *  However we want to disable the abortHandler when either of -Xrs / -Xrs:sync are present, so check for J9PORT_SIG_OPTIONS_REDUCED_SIGNALS_SYNCHRONOUS instead
+		/* Note that we need to do the same check in popDumpFacade().
+		 * Do not install the abort handler if -Xrs, -Xrs:sync or -XX:-HandleSIGABRT are present.
          */
```

Comment:
Minor thing but can you fix the formatting of this comment?  The closing `*/` doesn't line up correctly (and didn't in the original code either)